internal/pkg/describe/uri.go

```diff
@@ -230,7 +230,8 @@ func (d *uriDescriber) envDNSName(path string) (accessURI, error) {
 	if err != nil {
 		return accessURI{}, fmt.Errorf("get stack outputs for environment %s: %w", d.env, err)
 	}
-	if accessible, ok := envOutputs[envOutputPublicALBAccessible]; ok && accessible == "true" {
+	// Backward compatibility concern since envOutputPublicALBAccessible didn't exist before.
```

Comment:
nit: what do you think of
```go
if d.envDescriber.Version() < "v1.18.0" {}
```
to (perhaps, idk if it does) make it easier to understand for future?